cmd/machine-controller/main.go

```diff
@@ -64,6 +65,11 @@ func main() {
 		glog.Fatalf("Could not create client for talking to the apiserver: %v", err)
 	}
 
+	kubeClient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		glog.Fatalf("Could not create kubernetes client to talk to the apiserver: %v", err)
```

Comment:
This is not really `Fatal`. We could log at error and exit(1).

Reply:
Is not that the same?

Reply:
Well, if I run this from the command line I wasn't expecting a stack trace. Yes, print the error and exit. But the stacktrace makes it looks more fatal than it is.

Reply:
In that case we need to fix it everywhere. This is not the only place where the `Fatalf` is used.